lib/orbit-common/jsonapi-serializer.js

```diff
@@ -158,8 +158,10 @@ export default Serializer.extend({
           value = Object.keys(value).map(function(id) {
             return this.serializeRelationshipIdentifier(linkDef.model, id);
           }, this);
-        } else {
+        } else if (value) {
           value = this.serializeRelationshipIdentifier(linkDef.model, value);
+        } else {
+          value = null;
```

Comment:
Alternatively, I we make the serializeRelationshipIdentifer return null if !value --- what y'all thinkin?